tools/federation/src/main/java/org/eclipse/rdf4j/federated/FedX.java

```diff
@@ -113,16 +127,29 @@ public ValueFactory getValueFactory() {
 	public void initialize() throws SailException {
```

Comment:
`Sail#initialize` is deprecated, I think you should rename this to `Sail#init`.

Reply:
Good catch. I now make use of AbstractSail functionality and all the inherited base functionality. Required some more smaller adjustments, see last commit